packages/eslint-config-react-native/readme.md

```diff
@@ -0,0 +1,28 @@
+# @strv/eslint-config-react-native
+
+> STRV's ESLint config for React-Native projects
+
+Suitable for linting React Native source files. This ruleset extends `@strv/eslint-config-react` but does not extend `@strv/eslint-config-react/optional` nor `@strv/eslint-config-react/style`.
```

Comment:
Could you please add a _style.js_ and _optional.js_ files which extend their respective rulesets from the base one?

Check the `eslint-config-node` directory, this ruleset does exactly that.

The reasoning behind this is that you should be able to use all three variants from each platform-specific ruleset without having to know that there is some "base" ruleset from which you might also want to extend. So in a project, you would just do:

```js
extends = {
  '@strv/react-native',
  '@strv/react-native/optional',
  '@strv/react-native/style',
}
```

and that's it. ❤️ 🚀